Tests/Tests/RouterSpec.swift

```diff
@@ -291,7 +291,13 @@ class RouterSpec: QuickSpec {
                 }
 
                 it("will return false when tooltip identifier is incorrect") {
-                    expect(router.isDisplayingTooltip(with: "test")).to(beFalse())
+                    router.displayTooltip(tooltip, targetView: tooltipTargetView,
+                                          identifier: TooltipViewIdentifierMock,
+                                          imageBlob: imageBlob,
+                                          becameVisibleHandler: { _ in },
+                                          confirmation: true,
+                                          completion: { _ in })
+                    expect(router.isDisplayingTooltip(with: "incorrect_id")).to(beFalse())
```

Comment:
You need to wait for tooltip to appear first. i.e. put this `expect(window.findTooltipView()).toEventuallyNot(beNil())` before `expect()`